content/news/050/index.md

```diff
@@ -76,56 +101,39 @@ If needed, a section can be split into subsections with a "------" delimiter.
 [Way of Rhea][wor] just got a release date: it will be coming to Steam on
 **May 20th, 2024**!
 
-Way of Rhea is a color based puzzle game with difficult puzzles, but forgiving
-mechanics being developed by [@masonremaley] in a custom Rust engine.
+Way of Rhea ([Steam](https://store.steampowered.com/app/1110620/Way_of_Rhea/)) is a color based puzzle game with difficult puzzles, but forgiving
+mechanics being developed by [@masonremaley] in a custom engine.
```

Comment:
I think Rust isn't necessarily implied here, since some projects might only use Rust for part of the game. I think it would be good to keep the specifics in.

```suggestion
mechanics being developed by [@masonremaley] in a custom Rust engine.
```